### PR TITLE
All the changes needed for eas 3

### PR DESCRIPTION
**Pull request by palvarez89**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -6,15 +6,14 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "peru/ubuntu-18.04-server-amd64"
```
Comment (mariocj89) [inline Vagrantfile]: prod is still 14.0 isnt it?

File: roles/anxs.postgresql/.gitignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.DS_Store
```
Comment (mariocj89) [inline roles/anxs.postgresql/.gitignore]: should we add this as a submodule?

File: Vagrantfile (hunk 1 above)
Comment (palvarez89) [reply Vagrantfile]: Yes, but this won't go to current prod, ever

Comment (mariocj89) [reply Vagrantfile]: my worry is we are just testing for ubuntu18 and we might miss something. as soon as we can still deploy to prod, all good :)

File: roles/anxs.postgresql/.gitignore (hunk 2 above)
Comment (palvarez89) [reply roles/anxs.postgresql/.gitignore]: Hm.... That would make sense. I just wanted to not depend on 3rd party repos

Comment (mariocj89) [reply roles/anxs.postgresql/.gitignore]: Your call

File: roles/nginx/tasks/uwsgi.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
```
Comment (palvarez89) [inline roles/nginx/tasks/uwsgi.yml]: This is a bit overkill. Ubuntu provides Tha Mai package and the plugins needed. Uwsgi, uwsgi-plugin-python and uwsgi-plugin-python3

See https://packages.ubuntu.com/source/bionic/uwsgi

File: Vagrantfile (hunk 1 above)
Comment (palvarez89) [reply Vagrantfile]: if that ever happens, we could use the scripts from before these changes. Maintaining 14.04 and 18.04 is not going to be fun at all.

File: roles/nginx/tasks/uwsgi.yml (hunk 3 above)
Comment (palvarez89) [reply roles/nginx/tasks/uwsgi.yml]: sorted

File: roles/echaloasuerte-3/templates/echaloasuerte-uwsgi.ini [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+master = true
+processes = 5
+
+socket = {{ echaloasuerte_3_root }}/echaloasuerte-uwsgi{{ item }}.sock
+logto = {{ echaloasuerte_3_logs }}/echaloasuerte-uwsgi{{ item }}.log
+chmod-socket = 664
+vacuum = true
+
+env = DJANGO_SETTINGS_MODULE={{ echaloasuerte_3_app_settings }}
```
Comment (mariocj89) [inline roles/echaloasuerte-3/templates/echaloasuerte-uwsgi.ini]: Was it dns or DSN?

Comment (palvarez89) [reply roles/echaloasuerte-3/templates/echaloasuerte-uwsgi.ini]: Oops, I thought this was merged as DNS

https://github.com/etcaterva/eas-backend/blob/master/eas/settings/prod.py

**Pull request merged**